lib/sda/local.go

```diff
@@ -173,7 +179,11 @@ func (l *LocalTest) NewNode(tn *TreeNode, protName string) (*Node, error) {
 		TreeNodeID:   tn.Id,
 		RoundID:      uuid.NewV4(),
 	}
-	return NewNode(o, tok)
+	node, err := NewNode(o, tok)
+	if err == nil {
+		l.Nodes = append(l.Nodes, node)
+	}
+	return node, err
```

Comment:
This is the `Local`-module which is only to be used in tests. The case here is if we test a Node that is not bound to anything else. so the `Local`-module has to take care about this case itself.